chart/skywalking/templates/es-init.job.yaml

```diff
@@ -55,7 +55,11 @@ spec:
   {{- else }}
           value: "{{ .Values.elasticsearch.config.host }}:{{ .Values.elasticsearch.config.port.http }}"
 {{- end }}
+        - name: SW_ES_USER
```

Comment:
Need to determine whether these two keys exist, some users do not need to fill in these two keys. And you need to note that `elasticseach.config.xx` is currently only available to users who already have elasticseach.